webapp/components/sidebar_right_menu.jsx

```diff
@@ -264,6 +265,25 @@ export default class SidebarRightMenu extends React.Component {
                     </li>
                 );
             }
+
+            if (global.window.mm_config.EnableTeamCreation === 'true' || isAdmin) {
```

Comment:
@cpanato everything looks good to me except for the above permission in which we need to confirm.
@jwilander not really sure but should we use `isSystemAdmin` here for team creation instead of `isAdmin` only?